resources/language/German/strings.po

```diff
@@ -474,7 +474,7 @@ msgstr "Empfange Wiedergabefortschritt der Episoden von Trakt"
 
 msgctxt "#32120"
 msgid "Parsing episode playback progress %i of %i from Trakt"
-msgstr "Verarbeite Wiedergabefortschritt der Episoden %i of %i von Trakt"
+msgstr "Verarbeite Wiedergabefortschritt der Episoden %i von %i von Trakt"
```

Comment:
Well the second one changes the meaning unfortunately.

"Verarbeite Trakt Episoden Wiedergabefortschritt i% von i%"

I also just notice, that the meaning was already wrong, as `episode playback progress` is kind of a fixed term. Which I try to use to communicate what exactly is happening.